.travis.yml

```diff
@@ -109,6 +109,23 @@ matrix:
       env:
         - CLANG_CXX=clang++
         - DEPLOY=1
+
+    - os: linux
+      rust: stable
+      dist: bionic
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - python3-pip
+            - python3-setuptools
+            - python3-wheel
+      script:
+        - rustup component add rustfmt clippy
+        - pip3 install pre-commit
```

Comment:
Nit: use `--disable-pip-version-check --quiet --no-cache-dir` parameters to pip

Reply:
Done.